### PR TITLE
Prefix jQuery wrapped objects with $ in instructorFeedbacks.js #3798

### LINKED ISSUE #3798: Prefix jQuery wrapped objects with $ in instructorFeedbacks.js
Example: 
$currentRow = ("#modalTable).find('tr') 
instead of currentRow

### PR DESCRIPTION
**Pull request by junhaoyap**
Fixes #3798

### CODE REVIEW

**Comment by junhaoyap** [comment]
@taniach can you help look at this?

File: src/main/webapp/js/instructorFeedbacks.js [hunk 1]
```diff
@@ -135,16 +135,16 @@ function bindCopyButton() {
...
             return false;
         }
 
-        $(sessionsList).each(function() {
-            var cells = $(this).find('td');
-            var courseId = $(cells[0]).text();
-            var feedbackSessionName = $(cells[1]).text();
+        $sessionsList.each(function() {
+            var $cells = $(this).find('td');
+            var courseId = $($cells[0]).text();
```
Comment (taniach) [inline src/main/webapp/js/instructorFeedbacks.js]: Do we still need the brackets here?

Comment (junhaoyap) [inline src/main/webapp/js/instructorFeedbacks.js]: Which line in particular?

Comment (taniach) [inline src/main/webapp/js/instructorFeedbacks.js]: 146 does $cells[0].text() work?

Comment (junhaoyap) [inline src/main/webapp/js/instructorFeedbacks.js]: Negative, jQuery wrapped objects masquerades as an array but the objects retrieved from it are not jQuery wrapped objects - .text() is a jQuery function.

Comment (taniach) [inline src/main/webapp/js/instructorFeedbacks.js]: ok

Comment (junhaoyap) [inline src/main/webapp/js/instructorFeedbacks.js]: Thanks for the review!

**Pull request merged**